code/package-lock.json

```diff
@@ -1,8 +1,2169 @@
 {
   "name": "outsystems-datagrid-reactive",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I don't think we should be committing this file. What do you think, @rugoncalves ?

Reply:
That we should add this to .ignore file ;)

Reply:
I'll add it to the ignore file ;)